src/metabase/pulse/render/style.clj

```diff
@@ -11,7 +11,8 @@
      (style {:font-weight 400, :color \"white\"}) -> \"font-weight: 400; color: white;\""
   [& style-maps]
   (str/join " " (for [[k v] (into {} style-maps)
-                      :let  [v (if (keyword? v) (name v) v)]]
+                      :let  [v (if (keyword? v) (name v) v)]
+                      :when (not= (str v) "")]
```

Comment:
The idiomatic Clojure way to do this is what's called Nil Punning -- see https://guide.clojure.style/#nil-punning

Since we want `v` to be a string anyway and and `(name v)` returns a string I would just do it like this

```clj
:let  [v (if (keyword? v) (name v) (str v))]
:when (seq v)]
```

Reply:
Thanks for that! I was looking for a predicate that would cover both `nil` and `""` cases.

Reply:
You can also use `empty?` to check if something is empty, it basically the same as `(complement seq)`

Reply:
Yeah, my `(not= (str v) "")` was an attempt at getting to something like `(complement empty?)` :stuck_out_tongue_closed_eyes: